src/components/tooltip/Tooltip.tsx

```diff
@@ -19,14 +19,14 @@ interface TooltipProps {
   fontSize?: string;
   maxWidth?: string;
 }
-
 const Tooltip = (props: TooltipProps) => {
   const { text, children, fontSize, maxWidth } = props;
 
   const id = useId();
   const tooltipRef = useRef<HTMLDivElement>(null);
   const [show, setShow] = useState(false);
   const [position, setPosition] = useState({ top: 0, left: 0 });
+  const [isOverflowing, setIsOverflowing] = useState(false);
```

Comment:
```suggestion
  const [above, setAbove] = useState(false);
```
I think naming this `above` might make it clearer what it indicates -- that the tooltip is now positioned above the trigger, rather than to the side of it? Overflowing to me felt too ambiguous, in that it could still be overflowing while above the tooltip if it were still too wide etc.